testing/file_test/testdata/fail_example.carbon

```diff
@@ -3,5 +3,6 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 // AUTOUPDATE
-// CHECK:STDOUT: 2 args: `default_args`, `fail_example.carbon`
 // CHECK:STDERR: Oops
+
+// CHECK:STDOUT: 2 args: `default_args`, `fail_example.carbon`
```

Comment:
Just to note, the blank line between stderr and stdout here still seems odd to me. Neither have a line association so I would've expected them to be grouped together. Maybe a TODO?

Reply:
Maybe STDERR should just go at the end when there's no attached lines, same as STDOUT?

Anyways, not necessary to address here.